examples/counter/src/contract.rs

```diff
@@ -41,9 +42,6 @@ impl Contract for CounterContract {
         &mut self,
         value: u64,
     ) -> Result<ExecutionOutcome<Self::Message>, Self::Error> {
-        // Validate that the application parameters were configured correctly.
-        assert!(Self::parameters().is_ok());
```

Comment:
Should we call `let _ = self.runtime.application_parameters();`, like in crowd-funding?